trino-connector/integration-test/src/test/java/org/apache/gravitino/trino/connector/integration/test/TrinoQueryITBase.java

```diff
@@ -68,14 +68,17 @@ public class TrinoQueryITBase {
   protected static final String metalakeName = "test";
   protected static GravitinoMetalake metalake;
 
-  private static void setEnv() throws Exception {
+  private static BaseIT baseIT;
+
+  private void setEnv() throws Exception {
+    baseIT = new BaseIT();
```

Comment:
How to use `BaseIT`? What is the difference between creating an instance and inheritance? Can we add an abstract modifier to `baseIT` for uniform usage?

Reply:
> What is the difference between creating an instance and inheritance

Some tests, for example, `SparkQueryRunner` just use `BaseIT` as a field to start the `Gravitino` server and run TPCDS query.  Others just take it as the base test class. 

> Can we add an abstract modifier to baseIT for uniform usage?

it's beyond the scope of this PR scope and we can do the refacor later with the help of @FANNG1 @diqiu50 for the Spark and Trino related tests that use `BaseIT` as the class field.

Reply:
BaseIT is actually a utility for starting the Gravitino server and Gravitino client for TrinoTesters.  
In the TrinoQueryIT class, starting the Gravitino server is an option for running the tests. 
Therefore, it is not appropriate for the TrinoQueryIT class to inherit from BaseIT

Reply:
If refactoring is necessary, I think it would be better to move the logic for starting the Gravitino server out of BaseIT. In BaseIT, the automatic starting of the Gravitino server is implemented. Test classes that need to use the default method to start the Gravitino server should inherit from BaseIT

Reply:
Let me summarize the differences between the two usages:
Extending `BaseIT` automatically starts the Gravitino server and client before the test begins. If a developer wishes to start the Gravitino server manually, they should create a `BaseIT` instance, is that correct?

Reply:
I think so, refactoring this part will be another work for me.

Reply:
If that's the case, I believe adding comments to the class `BaseIT` is sufficient and there is no need to refactor it.